src/GraphQL/Instrumentation/ApolloTracingExtensions.cs

```diff
@@ -4,8 +4,20 @@
 
 namespace GraphQL.Instrumentation
 {
+    /// <summary>
+    /// Methods to add Apollo tracing metrics to an <see cref="ExecutionResult"/> instance.
+    /// </summary>
     public static class ApolloTracingExtensions
     {
+        /// <summary>
+        /// Adds Apollo tracing metrics to an <see cref="ExecutionResult"/> instance,
+        /// stored within <see cref="ExecutionResult.Extensions"/>["tracing"].
+        /// Requires that the GraphQL document was executed with metrics enabled;
+        /// see <see cref="ExecutionOptions.EnableMetrics"/>. With <see cref="InstrumentFieldsMiddleware"/>
+        /// installed, also includes metrics from field resolvers.
+        /// </summary>
+        /// <param name="result">An <see cref="ExecutionResult"/> instance.</param>
+        /// <param name="start">The UTC date and time that the GraphQL document began execution.</param>
         public static void EnrichWithApolloTracing(this ExecutionResult result, DateTime start)
```

Comment:
What do you think about such implementation?
```c#
public static void EnrichWithApolloTracing(this ExecutionResult result, DateTime start)
        {
            var perf = result?.Perf;
            if (perf != null)
                (result.Extensions ?? new Dictionary<string, object>())["tracing"] = CreateTrace(perf, start);
        }
```

Reply:
Apart from the fact that it requires `??=` instead of `??`, I'd say it looks good.  It operates exactly the same, so I'll include it in this pull request.

Reply:
Of course ??=. 🤦